config/internal.default.json

```diff
@@ -16,5 +16,13 @@
 		"logging": false
 	},
 	"payment_service": "http://localhost:5000",
-	"payment_complete_callback": "http://localhost:3001/v1/internal/payments"
+	"payment_complete_callback": "http://localhost:3001/v1/internal/payments",
+	"jwt": {
+		"private_keys": {
+			"default": {
+				"algorithm": "RS512",
```

Comment:
we decided to use `ES256`

Reply:
but you used RSA keys (the ones under `examples/`) so I just went with that.
also, why does it matter?

Reply:
I sent an article on slack about how JWT is broken and setting the algorithm is one way to prevent issues